pkg/agent/agent.go

```diff
@@ -313,9 +316,17 @@ func (i *Initializer) initOpenFlowPipeline() error {
 		// from local Pods to any Service address can be forwarded to the host gateway interface
 		// correctly. Otherwise packets might be dropped by egress rules before they are DNATed to
 		// backend Pods.
-		if err := i.ofClient.InstallClusterServiceCIDRFlows(i.serviceCIDR, gateway.MAC, gatewayOFPort); err != nil {
-			klog.Errorf("Failed to setup openflow entries for Cluster Service CIDR %s: %v", i.serviceCIDR, err)
-			return err
+		if i.serviceCIDR != nil {
+			if err := i.ofClient.InstallClusterServiceCIDRFlows(i.serviceCIDR, gateway.MAC, gatewayOFPort); err != nil {
```

Comment:
Do you think better to pass v4 and v6 CIDR together to InstallClusterServiceCIDRFlows()? Now it is the only case that a func is called separately for v4 and v6.